Formula/babl.rb

```diff
@@ -1,12 +1,16 @@
 class Babl < Formula
   desc "Dynamic, any-to-any, pixel format translation library"
   homepage "https://www.gegl.org/babl/"
-  url "https://download.gimp.org/pub/babl/0.1/babl-0.1.90.tar.xz"
-  sha256 "6e2ebb636f37581588e3d02499b3d2f69f9ac73e34a262f42911d7f5906a9243"
+  url "https://download.gimp.org/pub/babl/0.1/babl-0.1.92.tar.xz"
+  sha256 "f667735028944b6375ad18f160a64ceb93f5c7dccaa9d8751de359777488a2c1"
   license "LGPL-3.0-or-later"
   # Use GitHub instead of GNOME's git. The latter is unreliable.
   head "https://github.com/GNOME/babl.git", branch: "master"
 
+  # https://gitlab.gnome.org/GNOME/babl/-/merge_requests/45
+  # Can be removed on next version
+  patch :DATA
```

Comment:
Can we use the patch at https://gitlab.gnome.org/GNOME/babl/-/commit/b05b2826365a7dbc6ca1bf0977b848055cd0cbb6.diff?

Also, please use `brew style --fix babl` to fix the syntax errors.

Reply:
Done in 7c20d19.

Also done:

- `brew tests`
- `brew install --build-from-source babl`
- `brew test babl`
- `brew audit --strict --online babl`